frontend/src/pages/CoursesList/CoursesList.tsx

```diff
@@ -29,52 +30,47 @@ const CoursesList: FC<ICourseProps> = ({
   lastCourseRef,
   isAdmin,
 }) => (
-  <AuthorizedLayout pageName="Courses List">
+  <PageTitle title="Courses List">
```

Comment:
let's reuse title from constant?

Reply:
yes, we can, but if we will do this, we shold use this constants at route configs. We can do this inside other task.